.pre-commit-config.yaml

```diff
@@ -1,6 +1,7 @@
 # Install pre-commit hooks via `pip install pre-commit && pre-commit install`
 default_stages:
   - pre-push
+default_install_hook_types: [pre-push, pre-commit]
```

Comment:
by default if you run `pre-commit install` it will only install the `pre-commit` hooks

Reply:
good find!